docs/content/docs/flink/flink-getting-started.md

```diff
@@ -33,20 +33,20 @@ Iceberg integration for Apache Flink is available for Flink versions 1.12, 1.13,
 | [SQL create table like](#create-table-like)                 | ✔️     |                                                              |
 | [SQL alter table](#alter-table)                             | ✔️     | Only support altering table properties, column and partition changes are not supported |
 | [SQL drop_table](#drop-table)                               | ✔️     |                                                              |
-| [SQL select](#querying-with-sql)                            | ✔️     | Support both streaming and batch mode                        |
-| [SQL insert into](#insert-into)                             | ✔️ ️   | Support both streaming and batch mode                        |
+| [SQL select](#querying-with-sql)                            | ✔️     | Support both streaming and batch mode.                       |
+| [SQL insert into](#insert-into)                             | ✔️ ️   | Support both streaming and batch mode.                       |
 | [SQL insert overwrite](#insert-overwrite)                   | ✔️ ️   |                                                              |
 | [DataStream read](#reading-with-datastream)                 | ✔️ ️   |                                                              |
 | [DataStream append](#appending-data)                        | ✔️ ️   |                                                              |
 | [DataStream overwrite](#overwrite-data)                     | ✔️ ️   |                                                              |
-| [Metadata tables](#inspecting-tables)                       | ️      | Support Java API but does not support Flink SQL              |
+| [Metadata tables](#inspecting-tables)                       | ️      | Only support in Java API but not in Flink SQL.               |
```

Comment:
Nit:`Only supported with the Java API, not with Flink SQL.`

If you're updating the others, I'd update them as well to be more correctly phrased English sentences / phrases (so basically changing `only support` -> `only supports` or `only supported with` etc.